docs/basics/orders.rst

```diff
@@ -0,0 +1,522 @@
+######
+Orders
+######
+
+An order is a message passed into the 0x Protocol to facilitate an ERC20->ERC20 trade. There are currently two types of orders in 0x V4: Limit, RFQ.
+
+
+.. note::
+    As of v4 of the protocol, the maker address is no longer explicitly defined in limit orders. The maker is instead recovered from the signature of the order's EIP712 hash.
```

Comment:
Long shot necro-PR comment, forgive me! I would love to understand the design decision around dropping the `AssetProxy` concept from v4 and with it the ability to settle orders for NFTs - is there a paper trail for any discussion here?